config.go

```diff
@@ -95,6 +101,8 @@ func (c *Config) Validate() error {
 	switch {
 	case c.Group.Offsets.Retry.Max < 0:
 		return sarama.ConfigurationError("Group.Offsets.Retry.Max must be >= 0")
+	case c.Group.Offsets.Synchronization.DwellTime > time.Minute*10:
```

Comment:
really minor, but I think you should also add a `case c.Group.Offsets.Synchronization.DwellTime <= 0`